### PR TITLE
Fixed a typo, treshold to threshold.

### PR DESCRIPTION
**Pull request by CakHuri**

### CODE REVIEW

File: servers/physics/collision_solver_sat.cpp [hunk 1]
```diff
@@ -98,7 +98,7 @@ static void _generate_contacts_edge_edge(const Vector3 *p_points_A, int p_point_
 
 	Vector3 c = rel_A.cross(rel_B).cross(rel_B);
 
-	//if ( Math::abs(rel_A.dot(c) )<_EDGE_IS_VALID_SUPPORT_TRESHOLD ) {
+	//if ( Math::abs(rel_A.dot(c) )<_EDGE_IS_VALID_SUPPORT_THRESHOLD ) {
```
Comment (akien-mga) [inline servers/physics/collision_solver_sat.cpp]: It would be better to just remove this line.

File: scene/gui/gradient_edit.cpp [hunk 2]
```diff
@@ -236,8 +236,8 @@ void GradientEdit::_gui_input(const Ref<InputEvent> &p_event) {
 
 		//Snap to nearest point if holding shift
 		if (mm->get_shift()) {
-			float snap_treshhold = 0.03;
-			float smallest_ofs = snap_treshhold;
+			float snap_threshold = 0.03;
+			float smallest_ofs = snap_threshold;
```
Comment (toger5) [inline scene/gui/gradient_edit.cpp]: this is a little wiered anyways. `snap_thershold` does only get used there. 
maybe just do.
```
float smallest_ofs = 0.3; \\ acts as the snap threshold
```
or just rename `smallest_ofs` to `snap_threshold`

File: servers/physics/collision_solver_sat.cpp (hunk 1 above)
Comment (CakHuri) [reply servers/physics/collision_solver_sat.cpp]: Updated..

File: scene/gui/gradient_edit.cpp (hunk 2 above)
Comment (CakHuri) [reply scene/gui/gradient_edit.cpp]: There is something like : if (bla bla bla) { smallest_ofs = temp_ofs; ...... }

So, I'm not sure. ;-)

**Pull request merged**